src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -84,16 +84,15 @@ export class TechRecordSummaryComponent implements OnInit {
   }
 
   calculateVehicleModel(): void {
-    this.vehicleTechRecordCalculated = this.isEditable
-      ? { ...cloneDeep(this.vehicleTechRecord), reasonForCreation: '' }
-      : this.vehicleTechRecord;
+    this.vehicleTechRecordCalculated = this.isEditable ? { ...cloneDeep(this.vehicleTechRecord), reasonForCreation: '' } : this.vehicleTechRecord;
```

Comment:
Is it your auto-linter applying this change? They were put on separate lines for clarity.

Reply:
Yeah my auto-linter did do this, I can change it back if it's better for you?

Reply:
We have code formatters for a reason. If we all format code the same way then we'll stop getting unnecessary formatting changes. Manually formatting your code goes against the point of having formatters.